internal/testcli/golden_test.go

```diff
@@ -0,0 +1,13 @@
+package testcli
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSort(t *testing.T) {
```

Comment:
Isn't it better to test `PrepareReplacementsUser` instead?

Reply:
It's sort of tested via init_default_python.go

We might add separate test for it later if we see it makes sense.

Reply:
Tbh I just see very little value of testing `stableSortReverseLength` here